charts/ccd-data-store-api/values.yaml

```diff
@@ -36,6 +36,7 @@ java:
     ELASTIC_SEARCH_BLACKLIST: query_string
     ELASTIC_SEARCH_CASE_INDEX_NAME_FORMAT: '%s_cases'
     ELASTIC_SEARCH_CASE_INDEX_TYPE: _doc
+    ELASTIC_SEARCH_TIMEOUT: 6000
```

Comment:
can you rename this to `ELASTIC_SEARCH_REQUEST_TIMEOUT` please

Reply:
and let's set this to 10 sec please

Reply:
OK :)